### DISCUSSION TITLE
Can we have a Matrix room for Nickel?

### DISCUSSION THREAD

**Discussion by felschr** [tweag/nickel#1305] [Ideas]
Now, especially with the 1.0 release it would be nice to have a space to chat about Nickel.
I've also suggested this in the Nix Matrix Discussion room to be considered for the NixOS Space:
https://matrix.to/#/!GsmxjHfeAYLsTEQmjS:nixos.org/$YqvnvFFSj117TJHv1kHAzjDzxGR_s63MeWzhDX9Rr2o?via=nixos.org&via=matrix.org&via=nixos.dev

**Comment by matthew-healy** [comment]
There actually already is one, though it's currently not all that widely used: https://matrix.to/#/#nickel-lang:matrix.org

**Comment by felschr** [comment]
Oh, I couldn't find it via the room search in Element.
Looks like the rooms aren't published to matrix.org's public room directory.
Also the rooms just being called "General" & "Devs" would make finding them harder even if they were listed.

Perhaps it could also be mentioned on the website?

**Reply by matthew-healy** [reply]
Yeah, I also noticed it wasn't on the website or in the README while trying to find the link to share. If I get time I'll fix that later today. 

I'm not super familiar with how Matrix works, but I think listing & renaming the rooms sounds sensible. @yannham - is this something you're able to do? I assume it needs some sort of admin permissions.

**Reply by yannham** [reply]
What should we go with? Nickel Lang and Nickel Lang Dev, or something like that? Yes, we should mention them on the website. I'm opening an issue for that right now.

**Reply by matthew-healy** [reply]
Yeah, those names sound good to me!
